init.fish

```diff
@@ -3,6 +3,8 @@ set -g __dpaste_url_dpaste_de 'https://dpaste.de/api/'
 set -g __dpaste_keyword_dpaste_de 'content'
 set -g __dpaste_url_sprunge_us 'http://sprunge.us/'
 set -g __dpaste_keyword_sprunge_us 'sprunge'
+set -g __dpaste_url_gist_github_com 'https://api.github.com/gists'
+set -g __dpaste_keyword_gist_github_com 'github'
```

Comment:
I wanted to create the setup like this `set dpaste_site 'gist.github.com'`, similar to the existing way `__dpaste_send_url` and `__dpaste_keyword` are set.

I will then use `__dpaste_keyword` to check whether this is a gist